tests/pubsub/test_pubsub.py

```diff
@@ -510,3 +510,18 @@ def failed_sync_validator(peer_id, msg):
     await pubsubs_fsub[0].push_msg(pubsubs_fsub[0].my_id, msg_2)
     await asyncio.sleep(0.01)
     assert not event.is_set()
+
+
+@pytest.mark.parametrize("num_hosts, is_strict_signing", ((2, True),))
+@pytest.mark.asyncio
+async def test_strict_signing(pubsubs_fsub, hosts, monkeypatch):
```

Comment:
Would you mind adding a case which publishes messages with a {no signature|invalid signature}?